src/aiida/workflows/arithmetic/multiply_add.py

```diff
@@ -9,11 +9,9 @@
 # start-marker for docs
 """Implementation of the MultiplyAddWorkChain for testing and demonstration purposes."""
 
+from aiida.calculations.arithmetic.add import ArithmeticAddCalculation
 from aiida.engine import ToContext, WorkChain, calcfunction
 from aiida.orm import AbstractCode, Int
-from aiida.plugins.factories import CalculationFactory
-
-ArithmeticAddCalculation = CalculationFactory('core.arithmetic.add')
```

Comment:
mypy has troubles understanding that the CalculationFactory returns a Calcjob subclass in this case. So let's import it directly.

Reply:
yes had also this issue :/